florist/api/monitoring/metrics.py

```diff
@@ -0,0 +1,57 @@
+"""Classes for the instrumentation of metrics reporting from clients and servers."""
+import json
+from logging import DEBUG
+from typing import Any, Dict, Optional
+
+import redis
+from fl4health.reporting.metrics import DateTimeEncoder, MetricsReporter
+from flwr.common.logger import log
+
+
+class RedisMetricsReporter(MetricsReporter):  # type: ignore
+    """Save the metrics to a Redis instance while it records them."""
+
+    def __init__(
+        self,
+        redis_connection: redis.client.Redis,
+        run_id: Optional[str] = None,
+    ):
+        """
+        Init an instance of RedisMetricsReporter.
+
+        :param redis_connection: (redis.client.Redis) the connection object to a Redis. Should be the output
```

Comment:
I noticed you are using sphinx docstring format and I used the google format in the last PR. I don't have a strong preference and would be happy to adopt sphinx, do you think we should use the same format? Or does it not matter

Reply:
I just finished setting up automatic documentation with sphinx (following aieng-template changes) so I think we should stick to the sphinx format.

https://vectorinstitute.github.io/FLorist/reference/api/florist.api.launchers.launch.html

Reply:
Ahhhhh I seee, makes sense! I will keep that in mind going forward